cctools/as/i386.c

```diff
@@ -4804,7 +4805,7 @@ i386_operand (operand_string)
      char *operand_string;
 {
   const reg_entry *r;
-  char *end_op;
+    char *end_op = NULL;
```

Comment:
unnecessary whitespace added as indentation here?
```suggestion
  char *end_op = NULL;
```